common/app/common/NewNavigation.scala

```diff
@@ -379,16 +379,14 @@ object NewNavigation {
       }
     }
 
-    def getTopLevelSection(id: String) = {
-      val sectionList = sectionLinks.filter { item =>
+    def getParentPillar(id: String) = {
```

Comment:
It is good practice to specify the return type for every function. The compiler is smart enough to figure it out but it is easier for developer to understand what is this function doing by only looking at the full signature.